core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -686,6 +687,8 @@ public GraphHopper init( CmdArgs args )
 
         // routing
         defaultWeightLimit = args.getDouble("routing.defaultWeightLimit", defaultWeightLimit);
+        maxVisitedNodes = args.getInt("routing.maxVisitedNodes", Integer.MAX_VALUE);
```

Comment:
The default value can probably be written as `maxVisitedNodes` to follow current pattern.